java/arcs/core/storage/BUILD

```diff
@@ -49,6 +55,16 @@ kt_jvm_and_js_library(
     srcs = STORAGE_KEY_SRCS,
 )
 
+kt_jvm_and_js_library(
+    name = "reference",
```

Comment:
Nothing required for now, but if any more of these crop up, it might be worth putting them in a separate package. A pattern I've used in the past is to have an "api" subpackage with interfaces or things with no deps.

Reply:
Good idea, i'll remember that if we get more complexity here